URLs.md

```diff
@@ -0,0 +1,105 @@
+# Farcaster URLs
+
+Farcaster Messages may reference other messages, online content, or on-chain activity. An extension of the URL standard is employed by Farcaster to identify all resources and activities that can be referenced within protocol messages. 
+
+## Requirements
```

Comment:
I'd consolidate these and rename as "properties" or remove. They're helpful internally for constraining the spec, but I think we want to jump straight to the spec in the external-facing docs.